### PR TITLE
Week-13 Highlights

### PR DESCRIPTION
**Pull request by alphaComara**

### CODE REVIEW

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -3,7 +3,8 @@
 ## Overview
 
 What highlights can you come up with to share with the class?
-
```
Comment (rock43053) [inline HIGHLIGHTS.md:16]: Check

File: QUESTIONS.md [hunk 2]
```diff
@@ -1,5 +1,10 @@
 # Questions about Code, Node, Express, React
 
+1) In retrospect, I would like to review some of the higher order functions, and have more indeep coverage on this part.
+
 What questions can you come up with?
 
 ## Cohort Questions
+
+Are we getting any grades in order to determine how we are doing
```
Comment (jkearns885) [inline QUESTIONS.md:60]: I was wondering the same thing recently!

**Review by jkearns885 — COMMENTED** [review]
👍

File: QUESTIONS.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Questions about Code, Node, Express, React
 
+1) In retrospect, I would like to review some of the higher order functions, and have more indeep coverage on this part.
```
Comment (jessicareynolds) [inline QUESTIONS.md:3]: +1 I agree! This would be helpful.

File: QUESTIONS.md (hunk 2 above)
Comment (hollisjamison) [reply QUESTIONS.md:60]: Same! Ive been thinking of making a website that grabs student's repos from githubs and allows Stack to apply grades directly to github repos and allows students to login and see their grades, then donating it to Slack when I get done with the program. This is for sure an area of imporvement

**Pull request merged**